src/awscli_login/__init__.py

```diff
@@ -105,6 +105,12 @@ class Login(BasicCommand):
             'default': None,
             'help_text': 'HTTP Header to store the user\'s Duo passcode'
         },
+        {
+            'name': 'verify-ssl-certificate',
```

Comment:
I think the mis-match here with the version of this name with underscores may be causing the failing unit test.

```suggestion
'name': 'verify_ssl_certificate',
```

Reply:
Update: I'm pretty sure it's not the cause of the failure, but could still be worth a tweak for consistency.

Reply:
It should be hyphens on the CLI.